### PR TITLE
adding beneficiary basic

### PR DESCRIPTION
**Pull request by nabsrib**

### CODE REVIEW

File: cms/domain/beneficiary.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import dataclasses
+
+@dataclasses.dataclass
+class Beneficiary:
+    beneficiary_id: str #TODO- Currently nanoid, move to some ID class
+    fname: str
+    lname: str
+    mname: str
+    phone: str
+    email: str
```
Comment (ik-nitk) [inline cms/domain/beneficiary.py:10]: You can add govt_id and use cms.domain.id_type import IDType

File: development.md [hunk 2]
```diff
@@ -65,7 +65,11 @@ Running psql command inside database
 
 Insert some values into the member table
 ````
-insert into member (member_id, govt_id, id_type, fname, is_core, phone, email) values ('sample-111', 'gid-1111', 'AADHAAR' , 'sample' , false,  '99872000', 'email@sample')
+insert into member (member_id, govt_id, id_type, fname, is_core, phone, email) values ('sample-111', 'gid-1111', 'AADHAAR' , 'sample' , false,  '99872000', 'email@sample');
+
```
Comment (ik-nitk) [inline development.md]: add ```` to end the code block.  See Marckdown templates.

**Review by ik-nitk — COMMENTED** [review]
I guess we can add govt_id , use the same ID Type.

File: development.md (hunk 2 above)
Comment (nabsrib) [reply development.md]: ok done

File: cms/domain/beneficiary.py (hunk 1 above)
Comment (nabsrib) [reply cms/domain/beneficiary.py:10]: can we add govt id in next checkin? 
there were some sytacx issues when enum was referred for second time. 
I wanted to get 2nd table added to understand the code structure first.

**Pull request merged**